Dockerfile.minio

```diff
@@ -1,4 +1,4 @@
-FROM minio/minio:RELEASE.2022-05-26T05-48-41Z.hotfix.15f13935a
```

Comment:
What is the reason for this change?

Reply:
The hotfix container images are not multi-arch.

Reply:
I would like to update the version of Trino we use locally to more closely match what we use in production. I started making changes to get it working with a newer version but it was more involved.

Reply:
If you want a native image of Trino, just pull the repo and build the image. Let's not waste our time trying to make that a multi-arch image. It's not worth the effort until we decide to change the architecture of production.

There was a reason we were using the hotfix image. Instead of reverting to an old image, we should see if a new image works ok.

Reply:
It looks like we can use up to `RELEASE.2022-10-24T18-35-07Z` without much change. Starting with `RELEASE.2022-10-29T06-21-33Z`, the gateway and filesystem mode were removed, so existing local data needs to be migrated on the the `.trino/parquest_data` dir must be removed. I'm not sure how disruptive it is having to recreate/migrate. data stored in local buckets.